# this service wraps the transformPose method in TransformListener (both versions).
# if fixed_frame is empty the non-interpolating version is called and time is ignored.

string target_frame
time   target_time 
geometry_msgs/PoseStamped pin
string fixed_frame
---
geometry_msgs/PoseStamped pout